packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2547,6 +2590,9 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
   ///
   /// The default implementation mirrors the behavior of
   /// [visitChildren()] (which is supposed to walk all the children).
+  ///
+  /// The implementation has to return the children in paint order skipping all
+  /// children that are not relevant for semantics.
```

Comment:
These three paragraphs should be tweaked a bit, they're a bit repetitive right now.